R/google_drive.R

```diff
@@ -39,9 +40,22 @@ listItemsInGoogleDrive <- function(teamDriveId = NULL, path = NULL, type =  c("c
     if (pattern != "") {
       recursive <- TRUE
     }
+    if (isForSharedWithMe) {
+      if (pattern != "") {
+        pattern <- stringr::str_c("not 'me' in owners and ", pattern)
+      } else {
+        pattern <- "not 'me' in owners"
+      }
+    } else {
+      if (pattern != "") {
+        pattern <- stringr::str_c("'me' in owners and ", pattern)
+      } else {
+        pattern <- "'me' in owners"
+      }
+    }
     # To improve performance, only get id, name, mimeType, modifiedTime, size, parents for each file.
     # NOTE: googledrive changed team_drive argument to shared_drive
-    googledrive::drive_ls(path = path, type = type, shared_drive = teamDriveId, pageSize = 1000, fields = "files/id, files/name, files/mimeType, files/modifiedTime, files/size, files/parents, nextPageToken", n_max = n_max, q = pattern, recursive = recursive)
+    googledrive::drive_ls(path = path, type = type, shared_drive = teamDriveId, pageSize = 1000, fields = "files/id, files/name, files/mimeType, files/modifiedTime, files/size, files/parents, files/ownedByMe, files/owners/displayName, nextPageToken", n_max = n_max, q = pattern, recursive = recursive)
```

Comment:
Add comment on why we keep files/ownedByMe.

Reply:
Added comment.